### DISCUSSION TITLE
%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1130915%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

### DISCUSSION THREAD

**Discussion by giscus** [cit-hall-43/cit-hall-43.github.io#8] [Q&A]
# %E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1130915%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

讚美主

https://cit-hall-43.github.io/%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1130915%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1.html

**Comment by cit-hall-43** [comment]
『隨後，耶穌被那靈引到曠野，受魔鬼的試誘。』太四1
『祂禁食了四十晝四十夜，後來就餓了。』太四2
『那試誘者進前來，對祂說，你若是神的兒子，就叫這些石頭變成餅罷。』太四3
『耶穌卻回答說，經上記著，“人活著不是單靠食物，乃是靠神口裏所出的一切話。”』太四4

**Comment by cit-hall-43** [comment]
『身上肢體我們以為比較不體面的，就給它加上更豐盈的體面；我們不俊美的肢體，就得著更豐盈的俊美；』林前十二23
『至於我們俊美的肢體，就不需要了。但神將這身體調和在一起，把更豐盈的體面加給那有缺欠的肢體，』林前十二24
『免得身體上有了分裂，總要肢體彼此同樣相顧。』林前十二25
『若一個肢體受苦，所有的肢體就一同受苦；若一個肢體得榮耀，所有的肢體就一同歡樂。』林前十二26
『你們就是基督的身體，並且各自作肢體。』林前十二27

從《Android電子聖經》轉貼
http://twgbr.me
